docs/reference-architectures/containers/aks-pci/aks-pci-malware-content.md

```diff
@@ -216,7 +216,7 @@ Remove development, test and/or custom application accounts, user IDs, and passw
 
 ##### Your responsibilities
 
-As part of cluster creation, multiple local Kubernetes users are created by default. Those users can't be audited because they don't represent a unique identity. Some of them have high privileges. Disable those users by using the [Disable local accounts](/azure/aks/managed-aad#disable-local-accounts-preview) feature of AKS.
+As part of cluster creation, multiple local Kubernetes users are created by default. Those users can't be audited because they don't represent a unique identity. Some of them have high privileges. Disable those users by using the [Disable local accounts](https://learn.microsoft.com/en-us/azure/aks/manage-local-accounts-managed-azure-ad) feature of AKS.
```

Comment:
```suggestion
As part of cluster creation, multiple local Kubernetes users are created by default. Those users can't be audited because they don't represent a unique identity. Some of them have high privileges. Disable those users by using the [Disable local accounts](/azure/aks/manage-local-accounts-managed-azure-ad#disable-local-accounts) feature of AKS.
```